### ISSUE TITLE
Cannot load /usr/lib/apache2/modules/libphp7.1.so into server: /usr/lib/apache2/modules/libphp7.1.so: undefined symbol: ap_map_http_request_error

### ISSUE THREAD

**Issue by eguaj** [oerdnj/deb.sury.org#570]
This morning I upgraded my Ubuntu 14.04 VMs (running PHP 7.0 and PHP 7.1) and I stumbled upon this error message when restarting Apache :

```
# service apache2 restart
 * Restarting web server apache2
   ...fail!
 * The apache2 configtest failed.
Output of config test was:
apache2: Syntax error on line 140 of /etc/apache2/apache2.conf: Syntax error on line 3 of /etc/apache2/mods-enabled/php7.1.load: Cannot load /usr/lib/apache2/modules/libphp7.1.so into server: /usr/lib/apache2/modules/libphp7.1.so: undefined symbol: ap_map_http_request_error
Action 'configtest' failed.
The Apache error log may have more information.
```

This seems to affect the latest PHP 7.0 and PHP 7.1 (at least on Ubuntu 14.04):
- 7.0.17-1+deb.sury.org~trusty+4
- 7.1.3+-1+deb.sury.org~trusty+1

The only workaround I found was to revert `libapache2-php7.0` and `php7.0-common` to the previous working version:

```
# cd /var/cache/apt/archives
# dpkg -i libapache2-mod-php7.0_7.0.16-4+deb.sury.org~trusty+1_amd64.deb php7.0-common_7.0.16-4+deb.sury.org~trusty+1_amd64.deb
# service apache2 restart
```

**Comment by baurmatt** [comment]
Yep we too :(

**Comment by eguaj** [comment]
From PHP source code at https://github.com/php/php-src/blob/master/sapi/apache2handler/sapi_apache2.c#L65 I understand that the problem might be that the modules have been compiled against apache > 2.4.16 (which provides a "native" `ap_map_http_request_error`) while Ubuntu 14.04 is using apache 2.4.7 (and PHP sould be using the "workaround" version named `php_ap_map_http_request_error`).

**Comment by baurmatt** [comment]
@oerdnj: Could you please keep 1-2 old version in the repository? This would be really helpful for a fast recovery strategy.

**Comment by oerdnj** [comment]
@baurmatt No, launchpad doesn't support keeping old versions.

As a temporary fix add `ppa:ondrej/apache2` and upgrade apache2 as well.

**Comment by oerdnj** [comment]
Fixed in `7.0.17-2*` and `7.1.3+-2*` uploads.
